ipaclient/install/client.py

```diff
@@ -3713,6 +3714,9 @@ def __init__(self, **kwargs):
             raise RuntimeError(
                 "--ntp-pool cannot be used together with --no-ntp")
 
+        if self.request_cert:
+            logger.warning("Option --request-cert has been deprecated")
```

Comment:
Can you add "and will be removed in a future release" here as well?

Reply:
sure! It's done.